DataAccessLibrary/SqliteDataAccess.cs

```diff
@@ -50,7 +50,7 @@ public async Task<IEnumerable<(string, string, string, Dictionary<string, string
         public async Task<string> DbSave(string Table, string Column, string Value)
         {
             using IDbConnection connection = new SQLiteConnection(connectionString);
-            IEnumerable<string> output = await connection.QueryAsync<string>($"INSERT INTO {Table} ({Column}) VALUES ({Value})", new DynamicParameters());
+            IEnumerable<string> output = await connection.QueryAsync<string>($"INSERT INTO {Table} ({Column}) VALUES ({Value}) ON DUPLICATE KEY UPDATE ({Value})", new DynamicParameters());
```

Comment:
Er vi sikre på, at vi ALTID, hvis der er en konflikt, vil overskrive værdien i den kolonne?

Reply:
Konflikten sker hvis man prøver at insert data i en kombi der allerede eksistere, så skal den overskrive det.
AKA første gang en dag "scannes" skal den insert, næste gange den dag "scannes" skal den i stedet updateres.

Reply:
Ahh, så den sørger bare for at den samme værdi bliver brugt hver gang og at der ikke bliver oprettet flere unødvendige rækker i tabellen?

Reply:
yes